frontend/webpack.config.js

```diff
@@ -1,12 +1,14 @@
 const MiniCssExtractPlugin = require('mini-css-extract-plugin')
+const HtmlWebpackPlugin = require("html-webpack-plugin");
```

Comment:
Do we actually need to break out all the CSS files? Won't we get better performance leaving it bundled?

Reply:
Just a quick note that this isn't a new change -- it's been in master probably next to forever. 
As to whether or not a single file would lead to better performance, I'm not sure. I'll read up to see if there's preferred practice for this.

Reply:
Ah, didn't realize this wasn't new. Don't worry about it then.

Reply:
Just for clarity, we do the css splitting because it does improve the first load. The common styling is included in the main.css file. Specific css is loaded as needed. A single css would be better if the javascript were delivered as a single file.